snapshot/account.go

```diff
@@ -0,0 +1,97 @@
+// Modifications Copyright 2021 The klaytn Authors
+// Copyright 2019 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+//
+// This file is derived from core/state/snapshot/account.go (2021/10/21).
+// Modified and improved for the klaytn development.
+
+package snapshot
+
+import (
+	"bytes"
+	"math/big"
+
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/rlp"
+)
+
+/*
+  TODO-snapshot
```

Comment:
@jeongkyun-oh This means that you will modify it for Klaytn later? It seems that this will not be used for Klaytn.

Reply:
@kjhman21 Yes, I will update this to Klaytn on later commit. This is for the confirmation of the fact that the unittests are passed.